smogn/over_sampling.py

```diff
@@ -85,7 +85,7 @@ def over_sampling(
     num_dtypes = ["int64", "float64"]
     
     for j in range(d):
-        if data.iloc[:, j].dtype in num_dtypes and any(data.iloc[:, j] > 0):
+        if data.iloc[:, j].dtype in num_dtypes and all(data.iloc[:, j] > 0):
```

Comment:
What is this doing that `any()` is not?

Reply:
`all()` assures all elements are positive.
`any()` only assures that some elements are positive (there may also be negative elements).
I made this change because I thought the former is expected here.